Swiftfin tvOS/Views/LiveTVChannelsView.swift

```diff
@@ -11,6 +11,8 @@ import JellyfinAPI
 import SwiftUI
 import SwiftUICollection
```

Comment:
I've been looking at collection views for other work and remember you chose to use this one. I would prefer to move to https://github.com/apptekstudios/ASCollectionView as it's largely more supported and (seemingly) maintainable as the target source for this package is no longer active (and also weirdly a competitor)

If you feel like switching over to that collection view is a lot of work I'm fine to let this in, however it will be a goal to switch over to what I prefer both here and in tvOS.

Reply:
Understood, I can start looking into using `ASCollectionView` for Live TV stuff.

Reply:
Sorry for the delay on this. I am in progress on the updates. I have to finish out the change to ASCollectionView but I think I should have it ready soon.

Reply:
I thought I had already left a comment about this, but I can't seem to see it now- 

@LePips it seems `ASCollectionView` does not support tvOS: 
https://github.com/apptekstudios/ASCollectionView/issues/138


How would you prefer I proceed? 

Would you rather see `ASCollectionView` for iOS and `SwiftUICollection` for tvOS? or Keep them both on the same with `SwiftUICollection` for now?

Reply:
I think it's ideal to use SwiftUI's LazyVGrid.
Is there still a performance problem?

Reply:
Or we can look at SwiftUIX's CollectionView.
https://github.com/SwiftUIX/SwiftUIX/blob/95dfcbc789d7e04ee075b3b41296ee227aba6308/Sources/Intramodular/Collection%20View%20%26%20List/CollectionView.swift

Reply:
Well, bummer. Thanks for looking into that and I apologize if it took a good amount of time.

Overall, we need to be moving to a collection view for these large lists of objects (channels, libraries). LazyVGrid + ScrollView is a performance problem and is the cause of the stutters. It is lazy loaded, but it doesn't dequeue cells when necessary and that's what we need.

I loaded up ASCollectionView in a tvOS project and see that most of the errors are about tvOS versioning and some elements that I can remove (refresh control elements, table view stuff, keyboard stuff) if I were to fork it. I can try that sometime. Because of this thought, I'm more inclined towards ASCollectionView on iOS and SwiftUICollection on tvOS until I take a look at that, especially if you've already redone the work for iOS in ASCollectionView. Sure it'll be a big difference but we can just change it back or I'm honestly fine just leaving it if this doesn't pan out.

Overall, I just want to move away from the original source of the package which is a fork with a single change to the original repo, https://github.com/defagos/SwiftUICollection, which PangMo also linked. Do you need the change that our current source has made?

In the end, I'm fine with either option and I'll work with whatever you choose. I just hope Apple announces an official SwiftUI CollectionView tomorrow.

Reply:
Super cool, however I'd bet it will be iOS16+. What are your thoughts on that?

Reply:
Wait, so you're saying `SwiftUICollection` is a fork of the SwiftUIX collection view that PangMo linked? 

I'm not even aware of what change the fork made, I wouldn't anticipate any specific issues if we wanted to move to the SwiftUIX collection view for tvOS. 

We can also look to move towards Grid API depending on os level supported

Reply:
Oh sorry, I got my SwiftUICollection tabs mixed up. What I linked is the original SwiftUICollection and PangMo linked a different project. So again, you can make the decision to split or not.

Yes, the new Grid API will be iOS 16+, as well as the new NavigationStack that I want to use. We'll have this discussion elsewhere.

Reply:
looking at it again, I don't think I need the change that the ABJC fork added. However, I think I grabbed that SPM package by accident. I just tried again, look at this screenshot... even though I enter the `defagos` git URL, the one that Xcode's SPM tool seems to grab says ABJC. I'm not sure what is happening here..

<img width="839" alt="Screen Shot 2022-06-06 at 4 05 39 PM" src="https://user-images.githubusercontent.com/125045/172249404-0575bf22-36e4-4d34-bd51-4a8706c564d1.png">

Reply:
Ok, I think I won't have any issue switching to the original source of `SwiftUICollection` rather than the ABJC fork I grabbed. 

I think I'll make that adjustment so tvOS is still using `SwiftUICollection` for now, and we could consider SwiftUI Grid API as a possible future replacement.